nifi-h2/nifi-h2-database-migrator/src/test/java/org/apache/nifi/dbmigration/h2/TestH2DatabaseUpdater.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.dbmigration.h2;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestH2DatabaseUpdater {
+
+    public static final String AUDIT_DB_PATH = "./src/test/resources/nifi-flow-audit";
+    public static final String AUDIT_DB_PATH_FILE = AUDIT_DB_PATH + ".mv.db";
+    public static final String AUDIT_DB_PATH_FILE_ORIGINAL = "./src/test/resources/orig.nifi-flow-audit";
```

Comment:
Rather than adding two binary files to version control, it should be possible to create a `BeforeEach` annotated method that copies the `nifi-flow-audit` to a temporary directory.  Using the `@TempDir` annotation on a member variable of type `java.io.File` will cause JUnit 5 to create a temporary directory that can be used for migration operations.  This also should remove the need to manually removing backup files after test completion.